code/game/objects/items/kitchen.dm

```diff
@@ -141,7 +142,7 @@
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
 	desc = "A sharpened bone. The bare minimum in survival."
-	embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 35, "embedded_fall_chance" = 10)
+	embedding = list("embedded_pain_multiplier" = 2, "embed_chance" = 20, "embedded_fall_chance" = 10)

```

Comment:
was this meant to be 30 to match the others?

Reply:
its a piece of bone it should be less strong

Reply:
so it's just a nerf in a fix pr. kay.